open_instruct/dpo_tune_cache.py

```diff
@@ -602,11 +630,12 @@ def main(args: FlatArguments):
         dataset_args = {}
         if args.train_file is not None:
             data_files["train"] = args.train_file
-        raw_datasets = load_dataset(
-            args.train_file_type,
-            data_files=data_files,
-            **dataset_args,
-        )
+        with accelerator.main_process_first():
```

Comment:
Quick clarification question: don't we want to add `with accelerator.main_process_first()` on lines 612 and 621?